🐛 [Bug] Failed to build docker on tag=v1.0.0

##  Bug Description

Failed to build docker on tag=v1.0.0

## To Reproduce

Steps to reproduce the behavior:

1. git pull && git checkout v1.0.0
2. Use docker build command in docker/README.md, `docker build -f docker/Dockerfile -t torch_tensorrt1.0:latest .`
3. The error shows below

``` bash
...
Step 17/32 : RUN ./docker/dist-build.sh
 ---> Running in 5cf6bcd4cc13
No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda'                                                                                                                                                                                                          
running bdist_wheel
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
Loading:                                                                                                                                                                                                                                                             
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
    currently loading: 
Analyzing: target //:libtorchtrt (1 packages loaded, 0 targets configured)
Analyzing: target //:libtorchtrt (9 packages loaded, 14 targets configured)
Analyzing: target //:libtorchtrt (16 packages loaded, 43 targets configured)
Analyzing: target //:libtorchtrt (33 packages loaded, 78 targets configured)
Analyzing: target //:libtorchtrt (38 packages loaded, 591 targets configured)
Analyzing: target //:libtorchtrt (43 packages loaded, 2396 targets configured)
INFO: Analyzed target //:libtorchtrt (43 packages loaded, 2908 targets configured).
INFO: Found 1 target...
[0 / 1] [Prepa] BazelWorkspaceStatusAction stable-status.txt
[1 / 14] [Prepa] Symlinking virtual headers for //cpp:torch_tensorrt
[617 / 814] [Prepa] Symlinking virtual headers for @libtorch//:ATen
[1,061 / 1,173] Compiling cpp/bin/torchtrtc/main.cpp; 1s processwrapper-sandbox
[1,061 / 1,173] Compiling cpp/bin/torchtrtc/main.cpp; 11s processwrapper-sandbox
[1,061 / 1,173] Compiling cpp/bin/torchtrtc/main.cpp; 35s processwrapper-sandbox
[1,061 / 1,173] Compiling cpp/bin/torchtrtc/main.cpp; 43s processwrapper-sandbox
INFO: From Compiling cpp/bin/torchtrtc/main.cpp:
cpp/bin/torchtrtc/main.cpp: In function 'int main(int, char**)':
cpp/bin/torchtrtc/main.cpp:340:18: warning: catching polymorphic type 'class args::Help' by value [-Wcatch-value=]
  340 |   } catch (args::Help) {
      |                  ^~~~
cpp/bin/torchtrtc/main.cpp:343:29: warning: catching polymorphic type 'class args::ParseError' by value [-Wcatch-value=]
  343 |   } catch (args::ParseError e) {
      |                             ^
[1,062 / 1,173] [Scann] Compiling core/conversion/converters/impl/unary.cpp
[1,064 / 1,173] [Scann] Compiling core/conversion/conversion.cpp
[1,065 / 1,173] Compiling cpp/src/logging.cpp; 5s processwrapper-sandbox
[1,071 / 1,173] Compiling core/compiler.cpp; 6s processwrapper-sandbox
[1,077 / 1,173] Compiling core/conversion/converters/impl/reduce.cpp; 3s processwrapper-sandbox
[1,082 / 1,173] [Scann] Compiling core/ir/StaticParams.cpp
[1,085 / 1,173] Compiling core/lowering/passes/unpack_var.cpp; 5s processwrapper-sandbox
[1,090 / 1,173] Compiling core/lowering/passes/unpack_addmm.cpp; 1s processwrapper-sandbox
[1,092 / 1,173] Compiling core/conversion/converters/impl/interpolate.cpp; 21s processwrapper-sandbox
[1,095 / 1,173] Compiling core/lowering/passes/remove_bn_dim_check.cpp; 2s processwrapper-sandbox
[1,100 / 1,173] [Scann] Compiling core/conversion/converters/impl/conv_deconv.cpp
[1,105 / 1,173] Compiling core/lowering/lowering.cpp; 6s processwrapper-sandbox
[1,109 / 1,173] Compiling core/lowering/drop_unused_nodes.cpp; 1s processwrapper-sandbox
ERROR: /workspace/torch_tensorrt/src/core/lowering/BUILD:10:11: Compiling core/lowering/register_trt_placeholder_ops.cpp failed: (Exit 1): gcc failed: error executing command /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -g0 -O2 '-D_FORTIFY_SOURCE=1' -DNDEBUG -ffunction-sections ... (remaining 60 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
core/lowering/register_trt_placeholder_ops.cpp:16:34: error: invalid user-defined conversion from 'torch::jit::<lambda(torch::jit::Stack&)>' to 'torch::jit::OperationCreator' {aka 'std::function<void(std::vector<c10::IValue>*)> (*)(const torch::jit::Node*)'} [-fpermissive]
   16 |         aliasAnalysisFromSchema()),
      |                                  ^
core/lowering/register_trt_placeholder_ops.cpp:15:9: note: candidate is: 'torch::jit::<lambda(torch::jit::Stack&)>::operator void (*)(torch::jit::Stack&)() const' <near match>
   15 |         [](Stack& stack) { /*noop*/ },
      |         ^
core/lowering/register_trt_placeholder_ops.cpp:15:9: note:   no known conversion from 'void (*)(torch::jit::Stack&)' {aka 'void (*)(std::vector<c10::IValue>&)'} to 'torch::jit::OperationCreator' {aka 'std::function<void(std::vector<c10::IValue>*)> (*)(const torch::jit::Node*)'}
In file included from external/libtorch/include/torch/csrc/jit/runtime/custom_operator.h:5,
                 from core/lowering/register_trt_placeholder_ops.cpp:1:
external/libtorch/include/torch/csrc/jit/runtime/operator.h:91:24: note:   initializing argument 2 of 'torch::jit::Operator::Operator(std::string, torch::jit::OperationCreator, c10::AliasAnalysisKind)'
   91 |       OperationCreator op_creator,
      |       ~~~~~~~~~~~~~~~~~^~~~~~~~~~
Target //:libtorchtrt failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 469.309s, Critical Path: 57.93s
INFO: 1112 processes: 1072 internal, 40 processwrapper-sandbox.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
using CXX11 ABI build
building libtorchtrt
The command '/bin/sh -c ./docker/dist-build.sh' returned a non-zero code: 1
```

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,4 +1,4 @@
-ARG BASE=21.09
+ARG BASE=21.10
 ARG BASE_IMG=nvcr.io/nvidia/pytorch:${BASE}-py3
 FROM ${BASE_IMG} as base
 

